js-intro.js

```diff
@@ -2,61 +2,105 @@
 
 // Consider this variable:
 
-var mantra = "Be the dev";
+//var mantra = "B"
 
 // 1. Write the code that returns 'B' from mantra.
+   // console.log(mantra)
+
 
 // 2. Write the code that determines if there is a 'x' in mantra.
 
+//var mantra = "Be the Dev"
+//function mantra (){
```

Comment:
Good job so far! You are very close with this solution and the following one. A couple of tips that will help you moving forward are:
 1) get into the habit of naming things a bit more descriptively. In this case you want to distinguish your method from the variable that it works with
2) get into the habit of passing variables as arguments as opposed to manipulating them with a direct call inside the function.

so, try renaming the mantra function and refactor it to take a variable like mantra as an argument